packages/core/src/render3/styling_next/util.ts

```diff
@@ -225,6 +251,12 @@ export function getMapProp(map: StylingMapArray, index: number): string {
   return map[index + StylingMapArrayIndex.PropOffset] as string;
 }
 
+const MAP_DIRTY_VALUE = {};
```

Comment:
```suggestion
const MAP_DIRTY_VALUE = typeof ngDevMode !== 'undefined' && ngDevMode ? {}: {MAP_DIRTY_VALUE:true };
```
This makes debugging easier.

Reply:
Updated